### PR TITLE
added configFile argv for WCT.

### PR DESCRIPTION
**Pull request by hvdb**
see: https://github.com/Polymer/web-component-tester/pull/388

### CODE REVIEW

**Comment by FredKSchott** [comment]
Thanks @hvdb! I'll try to take a look at this and Polymer/web-component-tester#388 as soon as possible (The team is busy getting ready for Polymer Summit next week so it might be a little while before I can)

File: src/commands/test.ts [hunk 1]
```diff
@@ -104,6 +104,11 @@ export class TestCommand implements Command {
       description: "Set job visibility to 'public', 'public restricted', 'share', 'team' or 'private'",
       type: String,
     },
+    {
+      name: 'configFile',
```
Comment (FredKSchott) [inline src/commands/test.ts]: all cli flags have followed the `dash-case` format, please change to `config-file` and handle the mapping into WCT below.

Comment (hvdb) [inline src/commands/test.ts]: will do!

Comment (hvdb) [inline src/commands/test.ts]: The fix would be in WCT to specify the correct full name:
https://github.com/Polymer/web-component-tester/blob/master/runner/config.ts#L219

in addition i think the simpleOutput should also be fixed, here and in WCT

**Pull request closed without merging**